builtin/common/tests/serialize_spec.lua

```diff
@@ -133,7 +133,9 @@ describe("serialize", function()
 		end
 		local function num()
```

Comment:
This function is unused though, can't you just delete it?

Reply:
Huh? I don't think it's unused:

```lua
local primitives = {atomic, num, str}
local function primitive()
    return primitives[math.random(1, #primitives)]()
end
```
if it were unused, Luacheck would tell, letting the workflows fail.